go/libkb/env.go

```diff
@@ -924,6 +925,10 @@ func (e *Env) GetUpdateURL() string {
 	return e.config.GetUpdateURL()
 }
 
+func (e *Env) GetUpdateDisabled() (bool, bool) {
+	return e.config.GetUpdateDisabled()
```

Comment:
I only want to support a config setting, just for simplicity and since this code will be going away soon? (The other update env are only config settings too in this file, so this is consistent with those as well.)